github/resource_github_app_installation_repository.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"context"
+	"log"
+	"strconv"
+
+	"github.com/google/go-github/v32/github"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceGithubAppInstallationRepository() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubAppInstallationRepositoryCreate,
+		Read:   resourceGithubAppInstallationRepositoryRead,
+		Delete: resourceGithubAppInstallationRepositoryDelete,
```

Comment:
🤔 would we benefit from an `Update` function here as well? Or is creating a new App installation usually non-disruptive?